### ISSUE TITLE
MyBinder and Python 3 vs Python 3 (IBioIC)

### ISSUE THREAD

**Issue by peterjc** [widdowquinn/Teaching-IBioIC-Intro-to-Bioinformatics#53]
When running on MyBinder we got prompts about not finding a matching kernel, suggesting Python 2 or Python 3.

**Comment by widdowquinn** [comment]
Yep - I've made notes to add the option to the Dockerfile so that doesn't happen. My fault entirely.
